hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-de-1901.tex

```diff
@@ -1,18 +1,42 @@
-% dehypht-x-2014-05-21.pat
-
-\message{German Hyphenation Patterns (Traditional Orthography) `dehypht-x' 2014-05-21 (WL)}
-
-% TeX-Trennmuster für die traditionelle deutsche Rechtschreibung
-%
-%
-% Copyright (C) 2008, 2009, 2011, 2012, 2013, 2014 Werner Lemberg <wl@gnu.org>
-%
-% This program can be redistributed and/or modified under the terms
-% of the LaTeX Project Public License Distributed from CTAN
-% archives in directory macros/latex/base/lppl.txt; either
-% version 1 of the License, or any later version.
+% title: TeX-Trennmuster für die traditionelle deutsche Rechtschreibung
```

Comment:
Why in German? Shouldn't it be consistent with the rest and provide a nice English title? We can provide localised version in a separate field if needed, but I don't like this one.